config/locales/en.yml

```
@@ -1,8 +1,21 @@
 ---
 en:
+  account:
+    organisation_details_summary: If your organisation is not listed
+    organisation_text_html: |
+      <p>If you're from a central government organisation that publishes content on the GOV.UK website but your organisation is not listed, please %{contact_link}.</p>
```

Curly quote nitpick:
```suggestion
      <p>If you’re from a central government organisation that publishes content on the GOV.UK website but your organisation is not listed, please %{contact_link}.</p>
```